UI/users/preferences.html

```diff
@@ -80,17 +82,18 @@
                                   default_values = [ user.prefs.numberformat ],
                                   text_attr = 'format',
                                   value_attr = 'format',
+                                  required = 1
                                 } %]</td>
                           </tr>
                           <tr>
                             <th align="right"><label for="language">[% text('Language') %]</label></th>
-                            <td>[% PROCESS select_language element_data={
```

Comment:
Does this mean that the select_language dropdown should be replaced everywhere?

Reply:
Restored